src/core/p5.Graphics.js

```diff
@@ -59,6 +59,7 @@ p5.Graphics = function(w, h, renderer, pInst) {
   this._renderer._applyDefaults();
 
   this.name = 'p5.Graphics'; // for friendly debugger system
+  this._pInst = pInst;
```

Comment:
Isn't this attribute inherited from p5.Element? https://github.com/processing/p5.js/blob/970f6c14c6410205d10b82636b513e7eefd55cc2/src/core/p5.Element.js#L45